.github/workflows/up-board.yml

```
@@ -35,6 +35,40 @@ on:
         required: false
         type: string
         default: ''
+  # Allow this workflow to be called by other workflows
+  workflow_call:
+    inputs:
+      deploy-environment:
+        description: Environment to use for build and deploy
+        required: false
+        type: string
+        default: balena-staging.com
+      meta-balena-ref:
+        description: meta-balena ref if not the currently pinned version
+        required: false
+        type: string
+        default: ''
+      device-repo-ref:
+        description: device-repo ref if not the currently pinned version
+        required: false
+        type: string
+        default: master
+      test_matrix:
+        description: "JSON Leviathan test matrix to use for testing. No tests will be run if not provided."
+        required: false
+        type: string
+        default: ''
```

```suggestion
        default: '{}'
```
Allow this to evaluate as truthy so we don't need to set a default in meta-balena. It also needs to be fixed to akip an empty object on line 419 in balena-yocto-scripts.
https://github.com/balena-os/balena-yocto-scripts/blob/1cc4ee6817c0bc1ebac5eacad0e05f0bed28bb99/.github/workflows/yocto-build-deploy.yml#L419C13-L419C40